.github/workflows/im-services.yaml

```
@@ -51,3 +50,16 @@ jobs:
           docker build  -t selcohub/im-services-db:${{ env.VERSION }}-${{ env.commit_hash }} -f  Dockerfile .
           docker push selcohub/im-services-db:${{ env.VERSION }}-${{ env.commit_hash }}
         working-directory: backend/municipal-services/im-services/src/main/resources/db
+
+Workflow_Trigger:
```

actionlint (1.7.4)

unexpected key "Workflow_Trigger" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)